pkg/volumeclaimbinder/types.go

```diff
@@ -139,3 +145,65 @@ func matchStorageCapacity(pvA, pvB *api.PersistentVolume) bool {
 	bSize := bQty.Value()
 	return aSize <= bSize
 }
+
+// allPossibleMatchingAccessModes returns an array of AccessMode arrays that can satisfy a user's requested modes.
+// There are, essentially, 3 ways to request access modes:
+//
+//  RWO -- the most basic form of access (all storage volumes)
+//  RWO+ROX -- 1 writer or many read-only readers (GCEPersistentDisk)
+//  RWO+ROX+RWX -- many writers (NFS)
+//
+// A user's request can be satisfied with more modes, but never fewer.
```

Comment:
Same comment as above - do you expect this func to receive the requested mode: "I need at least ROX" or do you expect to receive the offered mode: "I can be used as ROX".  The former expands to [ ROX, RWX ], but the latter expands to [  RWO, ROX ].  From the call site this looks to be the offered mode, but the comment disagrees.